crescent/ext/tasks/task.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 from abc import ABC, abstractmethod
-from asyncio import TimerHandle, ensure_future, get_event_loop
+from asyncio import AbstractEventLoop, TimerHandle, ensure_future, get_event_loop
```

Comment:
Should this be done in a `typing.TYPE_CHECKING`?

Reply:
for now